test/require-config.js

```diff
@@ -6,30 +6,31 @@ require.config({
     baseUrl: '/base',
     waitSeconds: 60,
     paths: {
-        backbone: 'node_modules/backbone/backbone',
+        'backbone': 'node_modules/backbone/backbone',
```

Comment:
yep, we're using `consistent-as-needed` now, which means that because backbone.paginator needs quotes, all other properties of this object do too.